.github/workflows/continuous-integration.yml

```diff
@@ -13,12 +13,10 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.1"
-          - "7.2"
-          - "7.3"
-          - "7.4"
-          - "8.0"
           - "8.1"
+          - "8.2"
+          - "8.3"
+          - "8.4"
```

Comment:
Good new versions but library is used by 10% of php 7.4 and 8.0.
I may drop support only with major version increment and currently i have no reason to do this. 
If you return old versions i may merge this )

Reply:
The problem is that by keeping those old PHP versions we can't update other packages, it's a chicken and egg thing. So, you're gonna have to make a choice 🙃

Reply:
These are dev dependencies. is i understand some of them do not accept php 8.4? 
I will check this soon.
thanks

Reply:
Oh, never mind, it's actually working with PHP 7.4.

The problem is that I had to remove psalm, they're still working on it https://github.com/vimeo/psalm/issues/11107

Reply:
I couldn't make it work from version <=7.2